makr.go

```diff
@@ -88,7 +88,12 @@ func (g *Generator) Fmt(rootPath string) {
 	pwd, _ := os.Getwd()
 	files := []string{}
 	filepath.Walk(rootPath, func(path string, info os.FileInfo, err error) error {
-		path = strings.TrimPrefix(path, pwd+"/")
+		rel, err := filepath.Rel(pwd, path)
+		if err != nil {
+			rel = strings.TrimPrefix(path, pwd+string(filepath.Separator))
```

Comment:
Do you really need to keep this solution here, since it will produce a bad path? If `Rel` fails, it just means the path is invalid, so dropping the path makes sense.

Reply:
I'm fine with skipping the directory or file.

I tried to make the commits as simple as possible, such that it is clear that it doesn't behave worse than it did before.